Gemfile

```diff
@@ -223,11 +224,11 @@ group :development, :test do
   gem "bundler-audit", require: false
   gem "capybara"
   gem "minitest"
-  gem "minitest-rails"
+  gem "minitest-rails", github: "tsmartt/minitest-rails", branch: "rails71"
```

Comment:
Should this one live in `brave-intl` instead?

Reply:
Sure, I'll as IT to fork

Reply:
This is done https://github.com/brave-intl/minitest-rails/